http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterp.scala

```diff
@@ -90,23 +89,36 @@ final class LeoAppServiceInterp[F[_]: Parallel](
 
       clusterId = saveClusterResult.minimalCluster.id
 
-      //We want to know if the user already has an app to re-use that nodepool
-      userNodepoolOpt <- nodepoolQuery.getMinimalByUser(userInfo.userEmail, googleProject).transaction
+      machineConfig = req.kubernetesRuntimeConfig.getOrElse(
```

Comment:
I just realized a difference between the app and runtime APIs. For runtimes, we have a `RuntimeConfigRequest` type where each field is optional so we can have defaults for each individual field:
https://github.com/DataBiosphere/leonardo/blob/dfe49dfcf0291070ff04634ed760a09ceefe7dfb/core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/runtimeRoutesModels.scala#L34-L39

But for apps, the create request just takes a `Option[KubernetesRuntimeConfig]` where the individual fields are non-optional:
https://github.com/DataBiosphere/leonardo/blob/develop/core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala#L441

I don't think this needs to be changed right now (and the code in this PR is correct) -- it just made me pause for a minute while reviewing. :)